udata/commands/static.py

```diff
@@ -24,29 +24,45 @@
 def collect(path, input):
     '''Collect static files'''
     if exists(path):
-        print('"{0}" directory already exists and will be erased'.format(path))
+        msg = '"%s" directory already exists and will be erased'
+        log.warning(msg, path)
```

Comment:
Personnally, I prefer the `msg.format()` template syntax.
I that case, `logging` only support string interpolation with `%` syntax. The interpolation is lazy: no logger active, no interpolation done.
When you log a lot of line in debug, it avoid evaluating it if the logging level is `warning`.

I don't think this is critical here, but I'm used to use this syntax with logging when possible to prevent hidden performance loss.